util/dvsim/Deploy.py

```diff
@@ -341,6 +342,12 @@ def _set_attrs(self):
         self.pass_patterns = self.build_pass_patterns
         self.fail_patterns = self.build_fail_patterns
 
+        if hasattr(self.sim_cfg, "build_timeout_mins"):
+            self.timeout_mins = self.sim_cfg.build_timeout_mins
+        if self.timeout_mins:
+            log.log(VERBOSE, "Compile timeout for job \"%s\" is %d minutes",
+                    self.name, self.timeout_mins)
+
```

Comment:
Seems problematic: If the arg has no default value, and is not present at all we would get an error. If we had a default value we would need to figure out if the default value is due to the arg not being present, or because the user explicitly gave the argument and set it to the default value.

Reply:
If we did not pass `--build_timeout_mins`, then `self.deploy.args.build_timeout_mins` will still be present, and will be set to `None`.

Reply:
Done